### DISCUSSION TITLE
Do we need some more statistical knowledge in this project?

### DISCUSSION THREAD

**Discussion by Appointat** [ocademy-ai/machine-learning#115] [Ideas]
Title: Do we need some more statistical knowledge in this project?

Hello community,

I am considering submitting a discussion on the question of whether or not statistical knowledge is necessary for open-academy machine learning. As someone who has recently taken a course on statistics and has a more comprehensive understanding of the subject, I believe that statistical concepts can be very useful in certain areas of machine learning. However, I also recognize that my knowledge of statistics is more focused on the mathematical side and lacks practical applications in machine learning, for example, something like Suitability Testsor Independence Test.

I wanted to ask the community for their thoughts on this topic. In your experience, have you found statistical knowledge to be essential in your work with machine learning? What are some examples of statistical concepts or techniques that you have found to be particularly useful in your projects? On the other hand, have you encountered any limitations or drawbacks to relying too heavily on statistical knowledge in machine learning?

I am interested in hearing your perspectives on this topic and seeing if we can have a comprehensive discussion on the role of statistics in machine learning. Thank you in advance for your input!

Best regards,

**Comment by zhangqi444** [comment]
@Appointat Thanks for starting this discussion. 

Yes, statistics & mathematics are fundamental to Machine Learning. Basic stat & math knowledge will be great, actually necessary, to help you better understand how Machine Learning algorithms actually work.

We don't offer stat & math related content today. Because we mainly target students from STEM majors. We expect they have already taken or are in progress with stat & math courses in college. But we still cover the essential stat & math knowledge alongside with the Data Science and Machine Learning topics to explain the principles behind the algorithm. 

But you do raise a very good question here. The stat & math from school may not be enough or very good fitting to the DS/ML learning. It seems not to be a big problem for applied scientists from my work experience. But is definitely more important to people who are playing more research-focused roles, even in the industry. We will continue to collect feedback on this and prioritize stat & math content if needed. It may not come as separate sections first but could be added as an enhancement to the existing content. You are also welcome to file a PR if you'd like to contribute to this.
